docs/content/configuration/realtime.md

```diff
@@ -0,0 +1,78 @@
+---
```

Comment:
Is there anything linking to this page? It might make sense to put a reference to it in the consolidated config page and indicate that realtime nodes are deprecated.

Reply:
There wasn't anymore, I put a link to it at the end of the consolidated page and added a deprecation note